Feed back

Emmanuelaaron opened the pull request:
### What does this PR do
>implementing a better UI

### Tasks completed
>minimizing colors
>Increasing on the padding on the login form
>reducing color for the place holders
>Reducing on the width of the signup fields
>Getting a better background image

@nakimera @alexxsanya @bekeplar

Emmanuelaaron committed:
Feed back
###What does this PR do
implementing a better UI

###Tasks completed
minimizing colors
Increasing on the padding on the login form
reducing color for the place holders
Reducing on the width of the signup fields
Getting a better background image

nakimera commented:
Good job implementing the feedback @Emmanuelaaron 
The UI is much neater.
However, reduce the font sizes for the signup and logout links on the navigation bar
Lastly, for the logo, I would recommend you using one colour for EPIC and another colour for the logo so that it is not hard for a user to read

Emmanuelaaron committed:
Feedback
### What does this PR Do?
>implementing a better UI

### Tasks completed
>Reducing on the font size of logout/signout words
>Using a single color for the word epic and mail a different color

bekeplar commented:
I see you have done a great work on minimizing the colour. It's better now
